<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="CalculateMag" name="calculatemag"/>
      <widget class="AdgustMag" name="adgustMag"/>
      <widget class="SerialProt" name="serialPort"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <widget class="QMenu" name="menuEQF">
     <property name="title">
      <string>EQF</string>
     </property>
     <addaction name="actionimport"/>
     <addaction name="actionexport"/>
    </widget>
    <addaction name="actionNew_Import"/>
    <addaction name="menuEQF"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图(S)</string>
    </property>
    <addaction name="actionfreqChart"/>
    <addaction name="actionphaseChart"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>模式(P)</string>
    </property>
    <addaction name="actionGather"/>
    <addaction name="actionAdgust"/>
    <addaction name="actionSend"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionbnb"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>设置(I)</string>
    </property>
    <widget class="QMenu" name="menuSetStyle">
     <property name="title">
      <string>SetStyle</string>
     </property>
     <addaction name="actionOne"/>
     <addaction name="actiontwo"/>
    </widget>
    <addaction name="actionSmooth"/>
    <addaction name="menuSetStyle"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="actionfreqChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>freqChart</string>
   </property>
  </action>
  <action name="actionphaseChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>phaseChart</string>
   </property>
  </action>
  <action name="actionGather">
   <property name="text">
    <string>Gather</string>
   </property>
  </action>
  <action name="actionAdgust">
   <property name="text">
    <string>Adgust</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionSmooth">
   <property name="text">
    <string>Smooth</string>
   </property>
  </action>
  <action name="actionimport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionexport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionNew_Import">
   <property name="text">
    <string>New Import</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionbnb">
   <property name="text">
    <string>版本号</string>
   </property>
  </action>
  <action name="actionOne">
   <property name="text">
    <string>one</string>
   </property>
  </action>
  <action name="actiontwo">
   <property name="text">
    <string>two</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalculateMag</class>
   <extends>QWidget</extends>
   <header>Gather\calculatemag.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialProt</class>
   <extends>QWidget</extends>
   <header>Send\serialprot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdgustMag</class>
   <extends>QWidget</extends>
   <header>Adgust\adgustmag.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
